doc: gs: gnuarmemb path fix (#5346)
nrfconnect/sdk-nrf: doc/nrf/gs_installing.rst

#. Extract the contents of the root folder of the toolchain into a directory of your choice.
   The recommended folder is :file:`c:\gnuarmemb` on Windows and :file:`~/gnuarmemb` on Linux or macOS.
   Make sure that the folder name does not contain any spaces or special characters.

>    The contents are extracted to an additional folder whose name corresponds to the GNU Arm Embedded Toolchain version (*version-folder* in the following step).

Reviewer: ```suggestion
   The contents are extracted to another folder that corresponds to the GNU Arm Embedded Toolchain version (*version-folder* in the following step).
```

This is just a suggestion, not a required change. The original sentence felt a little bit long and complicated.
